.github/workflows/test.yaml

```diff
@@ -4,6 +4,33 @@ on:
   workflow_call:
 
 jobs:
+  inclusive-naming-check:
+    name: Inclusive naming
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: canonical/Inclusive-naming
+      - run: mv * /tmp
+      - uses: actions/checkout@v3
+      - name: Merge configuration files
+        run: |
+          # Combine all entries and replace matching elements by
+          # .name in .rules for the ones in .woke.yaml
+          if [ -f .woke.yaml ]; then
+            yq eval-all '
+            (
+              . as $item ireduce ({}; . *+ $item) | .rules | unique_by(.name)
+            ) as $mergedArray | . as $item ireduce ({}; . *+ $item) | .rules = $mergedArray
+            ' .woke.yaml /tmp/config.yml > /tmp/config.yaml
+          fi
+      - name: Run inclusive naming check
+        uses: canonical/inclusive-naming@main
+        with:
+          fail-on-error: "true"
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          reporter: github-pr-review
+          woke-args: '. -c /tmp/config.yaml'
```

Comment:
Will this not find anything in `.woke.yaml` and report it as a problem? (i.e. in that file we might have an override for "whitelist", but since we're running against the current directory would it not report that as a problem?)

Reply:
The file that's passed as an argument is the only one that is actually used. By default it will use the one defined in the action, which is the one that we copy to that location and override according to the contents of .woke.yaml

Reply:
The extension was different in both files, so it was not overwritten, but that's fixed now